Hymns/Browse/BrowseTab.swift

```diff
@@ -16,19 +16,19 @@ extension BrowseTab {
     var label: String {
         switch self {
         case .tags:
-            return "Tags"
+        return NSLocalizedString("Tags", comment: "")
```

Comment:
What is the "comment" used for?

Reply:
Ah, I see that it'll get auto-ported to the Localizable file. Can we go through and add them in to make it easy to translate? 😛 I'll help with it too when I get time  https://developer.apple.com/documentation/foundation/nslocalizedstring

Reply:
Do you mean add the same string to the comment? If that is what you mean I did that below.